src/app/clients/getClientsData.ts

```diff
@@ -41,44 +67,38 @@ const getClientsDataAndCount = async (
     const { data: clients, error: clientError } = await query;
 
     if (clientError) {
-        const logId = abortSignal.aborted
-            ? await logInfoReturnLogId("Aborted fetch: client table", clientError)
-            : await logErrorReturnLogId("Error with fetch: client table", clientError);
         if (abortSignal.aborted) {
-            throw new AbortError("fetch", "client table", "logId");
+            const logId = await logInfoReturnLogId("Aborted fetch: client table", clientError);
```

Comment:
(same in other places please)
```suggestion
            const logId = await logInfoReturnLogId("Aborted fetch: client table", { error: clientError });
```